src/reference-data-types.js

```diff
@@ -0,0 +1,43 @@
+module.exports = [
+  'AccountingReferenceType',
+  'AccountingRemarkType',
+  'Airport',
+  'AirAndRailMiscType',
+  'AirAndRailSupplierType',
+  'BusinessType',
+  //  'Carrier', //
+  'City',
+  'CityAirport',
+  'Country',
+  'Currency',
+  'EmailType',
+  //  'Equipment', //
+  'FulfillmentType',
+  //  'GeoPoliticalArea', //
```

Comment:
What is the purpose of having commented types in here?

Reply:
Well those are supported only through referenceDataSearch. Think it would be better to move into another file ?

Reply:
Will keep those in my backup dir till I complete integration of referenceDataSearch